central/sac/authorizer/builtin_scoped_authorizer.go

```diff
@@ -258,7 +258,8 @@ func (a *clusterNamespaceLevelScopeCheckerCore) SubScopeChecker(scopeKey sac.Sco
 }
 
 func errorScopeChecker(level interface{}, scopeKey sac.ScopeKey) sac.ScopeCheckerCore {
-	return sac.ErrorAccessScopeCheckerCore(errors.Wrapf(ErrUnexpectedScopeKey, "%T scope checked encountered %q", level, scopeKey))
+	utils.Should(errors.Wrapf(ErrUnexpectedScopeKey, "%T scope checked encountered %q", level, scopeKey))
```

Comment:
Why do you use `utils.Should` here but `utils.Must` above? Either is okay - `utils.Must` because both are programming errors, or `utils.Should` because whether they are triggered is not under control of this package -, but I don't see any reason to differentiate between the error causes.